content/accessible/labels-and-text-alternatives/index.md

```diff
@@ -0,0 +1,334 @@
+      </td>
+      <td>
+        <a href="#label-documents-and-frames">Label documents and frames</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        &lt;frame&gt; or &lt;iframe&gt; elements have a title
+      </td>
+      <td>
+        <a href="#label-documents-and-frames">Label documents and frames</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Image elements have [alt] attributes
+      </td>
+      <td>
+        <a href="#text-alternatives-for-images-and-objects">
+          Text alternatives for images and objects
+        </a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        &lt;input type="image"&gt; elements have [alt] text
+      </td>
+      <td>
+        <a href="#text-alternatives-for-images-and-objects">
+          Text alternatives for images and objects
+        </a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        &lt;object&gt; elements have [alt] text
+      </td>
+      <td>
+        <a href="#text-alternatives-for-images-and-objects">
+          Text alternatives for images and objects
+        </a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Buttons have an accessible name
+      </td>
+      <td>
+        <a href="#label-buttons-and-links">Label buttons and links</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Links have a discernible name
+      </td>
+      <td>
+        <a href="#label-buttons-and-links">Label buttons and links</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Form elements have associated labels
+      </td>
+      <td>
+        <a href="#label-form-elements">Label form elements</a>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Label documents and frames
+Every page should have a
+[`title`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title)
+element that briefly explains what the page is about. The `title` element gives
+the page its accessible name. When a screen reader enters the page, this is the
+first text that will be announced.
+
+<pre class="prettyprint devsite-disable-click-to-copy">
+&lt;!doctype html&gt;
+  &lt;html lang=&quot;en&quot;&gt;
+    &lt;head&gt;
+      <strong>&lt;title&gt;Mary's Maple Bar Fast-Baking Recipe&lt;/title&gt;</strong>
+    &lt;/head&gt;
+  &lt;body&gt;
+    …
+  &lt;/body&gt;
+&lt;/html&gt;
+</pre>
+
+<div class="aside note">
+For tips on writing effective titles, see the
+<a href="https://web.dev/discoverable/write-descriptive-text">
+Write descriptive titles guide
+</a>.
+</div>
+
+Similarly, any `frame` or `iframe` elements should have a `title` attribute.
+
+```html
+<iframe title="An interactive map of San Francisco" src="…"></iframe>
+```
+
+While an `iframe` may contain its own internal `title` element, a screen
+reader will usually stop at the frame boundary and announce the element's
+role—"frame"—and its accessible name, provided by the `title` attribute. This
+lets the user to decide if they wish to enter the frame or bypass it.
+
+## Text alternatives for images and objects
```

Comment:
Would it make sense to make this a call to action similar to other section titles? Include alt attributes for images and objects or something like that...